### PR TITLE
[Stable10] Fix path resolution for appdirs above ocwebroot

### PR DESCRIPTION
**Pull request by VicDeo**
Backport of https://github.com/owncloud/core/pull/30200

TL;DR

## Description
Allows app-theme to be in the directory located above oC webroot
Note: this app directory should be accessible from the web to serve js/css files properly

Checklist
- [x]  CSS
- [x] JS
- [x] templates
- [x] frontend translations *.js 
- [x] backend translations *.json 
- [x] Image path

## Related Issue
Fixes https://github.com/owncloud/core/issues/28778

## How Has This Been Tested?
Sample config:
```
  'apps_paths' => 
  array (
    0 => 
    array (
      'path' => '/var/www/oc-tmp/apps',
      'url' => '/apps',
      'writable' => false,
    ),
    1 => 
    array (
      'path' => '/var/www/appsinhell',
      'url' => '../appsinhell',
      'writable' => true,
    ),
  ),
```

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by PVince81** [comment]
Considering that this fixes a regression introduced by the theming code, backport is ok

**Review by PVince81 — APPROVED** [review]
:+1: assuming this is the same code (no conflicts/adjustments)
